web/Views/Landlord/Properties.cshtml

```diff
@@ -24,7 +24,8 @@
     }
     @foreach (var prop in Model.Properties)
     {
-        <tr>
+        <tr class ="clickable"
+            onclick="location.href = '@Url.Action("ViewProperty", "Property", new { propertyId = prop!.PropertyId })'">
```

Comment:
Unless it breaks the styling or something use an `<a>` tag around the table row rather than using javascript

Reply:
What are the reasons that we avoid Javascript wherever possible? Is it slower to be interpreted by a browser? Or is it more susceptible to errors or something, or for consistency?

Reply:
Using the elements built in to the browser over JS gives you all the built in behaviours e.g. you'll get a clicking hand cursor, and the link that would open will appear at the bottom of the page, and middle click to open in new tab will work, screen readers will have an easier time interpreting it.

You can kind of replicate all of those in CSS and JS, but it's more work and easy to get wrong. There's also some people that browse with JS off, though that's pretty uncommon these days.

Reply:
That's interesting, thank you! It makes sense

Reply:
Using the <a> tag currently destroys the styling, so I've left it how it is. If the tables are going to continue having clickable rows, I could raise a ticket to look into this (the Landlord List visible to admins currently uses the same JS, so it's not just this table which is affected)?

Reply:
That's fine, if it breaks the styling don't worry